### PR TITLE
'p' now pauses and unpauses the game

### PR DESCRIPTION
**Pull request by Philbywhizz**
Fixes issue #198 

A simple modification to allow the 'p' key to pause and unpause the game.

I had to remove the 'p' keydowns from the lander code as it was conflicting inputs with love.keypressed function.

Also factored out a CurrentScreenName() function. 

The more I look at it the more we really need a better FSM for managing game state.

### Note:
This PR is in draft mode pending #194 as there might need to be some rework.

### CODE REVIEW

**Comment by togfoxy** [comment]
Suggest you make this PR #4.  We have a nice queue forming.  :)

**Comment by togfoxy** [comment]
PR order:

- asset loader
- enet
- rangefinder fix
- p for pause

**Comment by togfoxy** [comment]
@Philbywhizz This is still draft. I'll let you resolve these conflicts while I work on the enet stuff.

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,14 +17,16 @@ The file format is based on [Keep a Change Log](https://keepachangelog.com/en/1.
...
+
 
 
 
 ### Fixed
 - fixed saved player name not being displayed correctly in-game
 - fixed regression introduced in v0.10 causing the rangefinder indicator not being displayed correctly
 - fixed the lag when running co-op. Can run two windows on localhost really well now (but only one at a time can receive keyboard input :( )
-- fixed a bizarre drawing bug when in co-op mode (work around appled)
+- fixed a bizarre drawing bug when in co-op mode (work around applied)
```
Comment (Philbywhizz) [inline CHANGELOG.md:29]: Just fixing a typo here retrospective

**Comment by Philbywhizz** [comment]
@togfoxy @MadByteDE 
I'm definately fighting against git here with trailing white space issues, etc. Rebase-ing my branch against main wasn't working so I had to merge main into my branch.

I'm happy with it now but perhaps another pair of eyes wouldn't hurt (it's been a long day)

**Review by togfoxy — APPROVED** [review]

**Pull request merged**